bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/database/ShadeCapabilitiesDatabase.java

```diff
@@ -41,17 +41,18 @@ public class ShadeCapabilitiesDatabase {
      */
     private static final Map<Integer, Capabilities> CAPABILITIES_DATABASE = Arrays.asList(
     // @formatter:off
-            new Capabilities(0).primary()                                                       .text("Bottom Up"),
-            new Capabilities(1).primary()        .tiltOnClosed()                                .text("Bottom Up Tilt 90°"),
-            new Capabilities(2).primary()        .tiltAnywhere().tilt180()                      .text("Bottom Up Tilt 180°"),
-            new Capabilities(3).primary()        .tiltAnywhere().tilt180()                      .text("Vertical Tilt 180°"),
-            new Capabilities(4).primary()                                                       .text("Vertical"),
-            new Capabilities(5)                  .tiltAnywhere().tilt180()                      .text("Tilt Only 180°"),
-            new Capabilities(6).primaryInverted()                                               .text("Top Down"),
-            new Capabilities(7).primary()                                 .secondary()          .text("Top Down Bottom Up"),
-            new Capabilities(8).primary()                                 .secondaryOverlapped().text("Dual Overlapped"),
+            new Capabilities( 0).primary()                                                       .text("Bottom Up"),
+            new Capabilities( 1).primary()        .tiltOnClosed()                                .text("Bottom Up Tilt 90°"),
+            new Capabilities( 2).primary()        .tiltAnywhere().tilt180()                      .text("Bottom Up Tilt 180°"),
+            new Capabilities( 3).primary()                                                       .text("Vertical"),
+            new Capabilities( 4).primary()        .tiltAnywhere().tilt180()                      .text("Vertical Tilt 180°"),
```

Comment:
Since this affects shade types, should this trigger any review/changes here? For example, in #12902 shade types 69-71 had capabilities changed from 3 to 4, but this is now indirectly reverted.

Reply:
Yes indeed. That is why it is on draft.

Reply:
FYI today's commit https://github.com/openhab/openhab-addons/pull/13330/commits/059ef09f4852810dba052c35444a20e4de1848cf encapsulates the state-of-the-art concerning my understanding..